### ISSUE TITLE
Add bulk inline editing to Table

### ISSUE THREAD

**Issue by mcarrano** [patternfly/patternfly#2319]
Adds bulk inline editing capability to a table. 

Design doc: https://docs.google.com/document/d/15aOWgpkd0Zp0hvbKM4jfG6Jf00FHjY_f92bkosbo1Ds/edit?usp=sharing

Design issue: https://github.com/patternfly/patternfly-next/issues/404

**Comment by mcarrano** [comment]
@rachael-phillips @mcoker FYI

**Comment by LHinson** [comment]
Putting this one on hold while we wait on  #2315 to complete

**Comment by mcarrano** [comment]
Will add this to the roadmap.

**Comment by mcoker** [comment]
@mcarrano should we leave this open?

**Comment by mcarrano** [comment]
Yes. We do have an approved design for this and therefore I'd like to keep it in the backlog. But we have not had any recent request.

**Comment by nicolethoen** [comment]
@mcarrano is the design for this feature still relevant/current? Should we add this to an upcoming milestone?

**Comment by mcarrano** [comment]
@nicolethoen I see this as a "nice-to-have" feature. No one is asking for it, specifically. What is the effort/risk to implement this? If you feel it can be easily implemented on top of existing row edit capability, I'd be interested in seeing a demo. Otherwise I can close.

**Comment by mcarrano** [comment]
That makes sense @nicolethoen . I have added the demo tag. Do you think we need a core demo of this or can this go directly to React? Not sure what the core demo would be.

**Comment by mcoker** [comment]
@mcarrano we can do it as a core demo to make sure it works and show the structure.
